html_utils/scraping.py

```diff
@@ -0,0 +1,11 @@
+import requests
+from bs4 import BeautifulSoup
+
+HTML_PARSER = "html.parser"
+
+def do_get_request(url: str) -> requests.Response:
+    response = requests.get(url)
+    return response
+
+def get_soup_for_html(html_payload: str or bytes, html_parser: str = HTML_PARSER) -> BeautifulSoup:
```

Comment:
missing docstrings

Reply:
why do you think you need a separate method for this call?

Reply:
I'll add it to the view logic.

Reply:
done